CheckDrive.Mobile/CheckDrive.Mobile/CheckDrive.Mobile/App.xaml.cs

```diff
@@ -25,7 +25,11 @@ public App()
 
             ConfigureServices();
 
+            Connectivity.ConnectivityChanged += Connectivity_ConnectivityChanged;
+
             MainPage = new AppShell();
+
+            //var signalRService = new SignalRService();
```

Comment:
will this commentary be needed later?

Reply:
No, it is not needed. It is for testing Popup page, because Popup page will work runtime.